README.md

```diff
@@ -388,10 +389,13 @@ You can integration this plugin with ```fastify-helmet``` with some little work.
   return {
     contentSecurityPolicy: {
       directives: {
-        defaultSrc: ["'self'"], // default source is mandatory
-        imgSrc: ["'self'", 'data:', 'validator.swagger.io'],
-        scriptSrc: ["'self'"].concat(instance.swaggerCSP.script),
-        styleSrc: ["'self'", 'https:'].concat(instance.swaggerCSP.style)
+					...helmet.contentSecurityPolicy.getDefaultDirectives(),
+					"form-action": ["'self'"],
+					"img-src": ["'self'", "data:", "validator.swagger.io"],
+					"script-src": ["'self'"].concat(instance.swaggerCSP.script),
+					"style-src": ["'self'", "https:"].concat(
+						instance.swaggerCSP.style
+					),
```

Comment:
why this style change?

Reply:
Hi @mcollina, has to be in kebab-case unfortunately, see https://github.com/fastify/fastify-helmet/pull/109#issuecomment-759411530

Reply:
I mean, the alignment is all wrong.

Reply:
Sorry ended up mixing tabs with the spaces :confounded: , fixed in 6fae0e1